### ISSUE TITLE
onImageSelected not working when select a image from camera

### ISSUE THREAD

**Issue by guuilp** [ParkSangGwon/TedBottomPicker#64]
Getting a image from gallery is working fine, but when I took a photo from camera it does not go to onImageSelected method. Can you help me?

```
                TedBottomPicker tedBottomPicker = new TedBottomPicker.Builder(CreateGroupActivity.this)
                        .setOnImageSelectedListener(new TedBottomPicker.OnImageSelectedListener() {
                            @Override
                            public void onImageSelected(Uri uri) {
                                beginCrop(uri);
                            }
                        })
                        .create();

                tedBottomPicker.show(getSupportFragmentManager());
```

**Comment by ParkSangGwon** [comment]
Did you see logcat?

**Comment by Nonisharma** [comment]
I am facing same problem please ans

**Comment by Nonisharma** [comment]
you need to add the if else concditon  in TedBottomPicker.java file please follow my code

 if (cameraInent.resolveActivity(getActivity().getPackageManager()) == null) {
            errorMessage("This Application do not have Camera Application");
            return;
        }else {
            ContentValues values = new ContentValues(1);
            values.put(MediaStore.Images.Media.MIME_TYPE, "image/jpg");

            File imageFile = getImageFile();
            Uri photoURI = FileProvider.getUriForFile(getContext(), getContext().getApplicationContext().getPackageName() + ".provider", imageFile);

            List<ResolveInfo> resolvedIntentActivities = getContext().getPackageManager().queryIntentActivities(cameraInent, PackageManager.MATCH_DEFAULT_ONLY);
            for (ResolveInfo resolvedIntentInfo : resolvedIntentActivities) {
                String packageName = resolvedIntentInfo.activityInfo.packageName;
                getContext().grantUriPermission(packageName, photoURI, Intent.FLAG_GRANT_WRITE_URI_PERMISSION | Intent.FLAG_GRANT_READ_URI_PERMISSION);
            }

            cameraInent.putExtra(MediaStore.EXTRA_OUTPUT, photoURI);
            startActivityForResult(cameraInent, REQ_CODE_CAMERA);
        }

**Comment by kickstartinteractiveandroidteam** [comment]
i face this problem only when the activty have onActivityResult so use
 @Override
    public void onActivityResult(int requestCode, int resultCode, Intent intent) {
        super.onActivityResult(requestCode, resultCode, intent);

**Comment by ParkSangGwon** [comment]
This issue solved at `v1.2.0`.
I use `TedRxOnActivityResult` instead of `startActivityForResult()`.
https://github.com/ParkSangGwon/TedRxOnActivityResult
